backend/test/akvo/lumen/lib/user_test.clj

```diff
@@ -0,0 +1,160 @@
+(ns akvo.lumen.lib.user-test
+  (:require [akvo.lumen.component.emailer :as emailer]
+            [akvo.lumen.component.keycloak :as keycloak]
+            [akvo.lumen.component.tenant-manager :refer [pool]]
+            [akvo.lumen.fixtures :refer [migrate-tenant rollback-tenant]]
+            [akvo.lumen.lib :as lib]
+            [akvo.lumen.lib.user :as user]
+            [akvo.lumen.variant :as variant]
+            [clojure.edn :as edn]
+            [clojure.java.io :as io]
+            [clojure.pprint :refer [pprint]]
+            [clojure.set :as set]
+            [clojure.test :refer :all]
+            [com.stuartsierra.component :as component]
+            [duct.component.hikaricp :refer [hikaricp]]
+            [ragtime.jdbc :as rjdbc]
+            [ragtime.repl :as rrepl]))
+
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;; System setup
+;;;
+
+(def seed-data
+  (->> "test-seed.edn" io/resource slurp edn/read-string))
+
+(def t1 (first (filter #(= "t1" (:label %))
```

Comment:
I was actually wondering if we even need the second tenant at all. Right now we have no tests or anything that use it.